### PR TITLE
Asset Database documentation. Overview of what the Asset Database is,…

### PR DESCRIPTION
**Pull request by AMZN-stankowi**
## Change summary

Added a page describing the Asset Database, and how to make changes to the Asset Database. Some of this was originally written by [amzn-mike](https://github.com/amzn-mike).

### Submission Checklist:

* [ ] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [ ] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [ ] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [ ] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Review by micronAMZN — CHANGES_REQUESTED** [review]

File: content/docs/user-guide/assets/asset-processor/asset-database.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+
+# Updating Asset Processor & the Asset Database
+
+There are two primary workflows for interacting with the **Asset Database** by making changes to **Asset Processor** code. [Adding a new query,](#NewQuery) and [making changes to the Asset Database.](#UpdateDatabase)
+
+## Adding a New Query {#NewQuery}
+
+**Asset Database** queries are defined in [AssetDatabaseConnection.cpp](https://github.com/o3de/o3de/blob/development/Code/Framework/AzToolsFramework/AzToolsFramework/AssetDatabase/AssetDatabaseConnection.cpp#L49).
+
+### 1. Create Static Const Strings for the Query
```
[the remark this answers is not in the document]
Comment (AMZN-stankowi) [reply content/docs/user-guide/assets/asset-processor/asset-database.md:55]: That didn't work well when I tried to do the code blocks in the tutorial. Also, the nested steps don't seem to work well with that setup. Any idea on how to make that work?

With that change:
image


Without, as I had it originally:
image

File: content/docs/user-guide/assets/asset-processor/asset-database.md [hunk 2]
```diff
@@ -0,0 +1,197 @@
+---
+linkTitle: Asset Database
+title: Asset Database
+description: The Asset Database is used by the Asset Processor to track the work it has completed.
+weight: 200
```
[the remark this answers is not in the document]
Comment (AMZN-stankowi) [reply content/docs/user-guide/assets/asset-processor/asset-database.md]: Fixed

File: content/docs/user-guide/assets/asset-processor/asset-database.md [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+toc: true
+---
+
+The **Asset Database** is an SQLite database used by the [**Asset Processor**](docs/user-guide/assets/asset-processor) to track the work it has completed. The **Asset Database** is not used by the Editor or Launchers, which instead use the **Asset Catalog** to track information about available assets.
+
+The **Asset Database** is located in the [**Asset Cache**](docs/user-guide/assets/pipeline/asset-cache) for a project, in assetdb.sqlite.
```
[the remark this answers is not in the document]
Comment (AMZN-stankowi) [reply content/docs/user-guide/assets/asset-processor/asset-database.md]: Fixed

File: content/docs/user-guide/assets/asset-processor/asset-database.md [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+# Asset Database Overview
```
[the remark this answers is not in the document]
Comment (AMZN-stankowi) [reply content/docs/user-guide/assets/asset-processor/asset-database.md]: Fixed

**Review by FiniteStateGit — APPROVED** [review]

**Review by micronAMZN — APPROVED** [review]

**Pull request merged**